arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -496,5 +496,5 @@ bool ClusterIndex::supportsStreamInterface(
     default:
       break;
   }
-  return false;
+  return {};
```

Comment:
I would prefer to be more explicit:
```suggestion
  return Index::StreamSupportResult::makeUnsupported();
```